src/ClientData/ScopeStats.cpp

```diff
@@ -8,13 +8,15 @@
 
 namespace orbit_client_data {
 void ScopeStats::UpdateStats(uint64_t elapsed_nanos) {
+  double old_avg = static_cast<double>(average_time_ns());
   count_++;
   total_time_ns_ += elapsed_nanos;
-  uint64_t old_avg = average_time_ns();
+  double new_avg = static_cast<double>(average_time_ns());  // it should be double for arithmetics
```

Comment:
This is a bit out of the scope of this pull request, but now that the average time is not stored but computed, `average_time_ns()` should instead be `ComputeAverageTimeNs()`. The style guide only allows snake_case for basic accessor that correspond to the name of a member variable. This stands out here because if `average_time_ns()` was a simple accessor, `count_++; total_time_ns_ += elapsed_nanos;` wouldn't cause it to change.

Reply:
Good point. As I introduces this. I will tackle it.